src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -1430,6 +1473,26 @@ private static Set<String> validateTableProperties(Set<String> properties, Strin
         return lowerCaseProperties;
     }
 
+    private static Set<String> validateKeyspaceProperties(Set<String> properties, String name)
+    {
+        if(properties == null)
+            throw new IllegalArgumentException(format("Invalid value for %s: null is not allowed", name));
+        
+        Set<String> lowerCaseProperties = properties.stream().map(String::toLowerCase).collect(toSet());
+
+        Set<String> validKeyspaceProperties = Set.of(
+                                                KeyspaceParams.Option.DURABLE_WRITES.toString(),
+                                                KeyspaceParams.Option.REPLICATION.toString(),
+                                                KeyspaceParams.Option.FAST_PATH.toString()
+                                            );
+        Set<String> diff = Sets.difference(lowerCaseProperties, validKeyspaceProperties);
+
+        if(!diff.isEmpty())
+            throw new IllegalArgumentException(format("Invalid value for %s: '%s' does not parse as valid keyspace property", name, diff));
```

Comment:
`diff` can contain multiple items, it should be `Invalid values for ...` and `... '%s' do not parse as valid keyspace properties`. Same as for table case.